h2o-genmodel/src/main/java/hex/genmodel/algos/gam/GamMojoReader.java

```diff
@@ -22,7 +22,6 @@ protected void readModelData() throws IOException {
     _model._family = DistributionFamily.valueOf((String)readkv("family"));
     _model._cats = readkv("cats", -1);
     _model._nums = readkv("num");
-    _model._numsCenter = readkv("numsCenter");
```

Comment:
should we also remove from writer since no woriies abou bacwards compatibility (this is the first release of gam mojo)

Reply:
Oh sure !